.github/workflows/FuseQuery-Coverage.yml

```diff
@@ -19,7 +19,7 @@ jobs:
         uses: actions-rs/toolchain@v1
         id: toolchain
         with:
-          toolchain: stable
+          toolchain: nightly
```

Comment:
Why we use nightly here?

Reply:
Because `actions-rs/toolchain@v1`  only works in nightly.